pulsar-discovery-service/src/main/java/org/apache/pulsar/discovery/service/server/ServiceConfig.java

```diff
@@ -75,6 +75,16 @@
     // Authorization provider fully qualified class-name
     private String authorizationProvider = PulsarAuthorizationProvider.class.getName();
 
+    public int getMaxMessageSize() {
+        return maxMessageSize;
+    }
+
+    public void setMaxMessageSize(int maxMessageSize) {
+        this.maxMessageSize = maxMessageSize;
+    }
+
+    private int maxMessageSize = 5242880;
```

Comment:
This should be defined in a static constant, shared between client and broker.

Reply:
will fix